docs/reference/rest-api/security/get-builtin-privileges.asciidoc

```diff
@@ -92,6 +92,7 @@ A successful call returns an object with "cluster", "index", and "remote_cluster
     "manage_rollup",
     "manage_saml",
     "manage_search_application",
+    "manage_search_connector",
```

Comment:
FYI this will be in docs.

Reply:
Oh right, it's also `.asciidoc` file! Let me remove it